packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import UpdateCypressModal from './UpdateCypressModal.vue'
+import { defaultMessages } from '@cy/i18n'
+
+describe('<UpdateCypressModal />', { viewportWidth: 1000, viewportHeight: 750 }, () => {
+  it('renders expected content & emits expected events', () => {
+    const installedVersion = '8.2.0'
+    const latestVersion = '10.0.0'
+
+    cy.mount({
+      render: () => <UpdateCypressModal installedVersion={installedVersion} latestVersion={latestVersion} show={true}></UpdateCypressModal>,
```

Comment:
Why are we doing this with render? Our other tests do 

```ts
cy.mount(() => <Comp ... />
```

Also I think we can write this a bit more concisely/idiomatically (we should probably figure out if we want to keep the `vueWrapper` API at all).

```ts
const onClose = cy.stub()
cy.mount(() => <UpdateCypressModal onClose={onClose} />) 

cy.findByLabelText('Close').click().then(() => {
      expect(onClose).to.have.been.calledWith([])
    })
```

Something like this should work. A bit of a trick but if you are doing `@close` it is compiled into `onClose`, so this works.

Reply:
I think was trying to be consistent with our `mountFragment` tests, I'm fine removing `render` here.

And yeah let's talk about the approach to events. I'm not sure I find the stub method more concise as such, but I like that it doesn't get into framework specific details, and if we are consistent with how we validate that expected events are firing, that's great. Though in this example I don't think it would actually be called with the empty array? We'd just expect it to be called period.

I haven't run into other situations where I want the `vueWrapper` API personally, but I have seen a lot of code that as part of testing a component in jest, will reach into it and basically unit test specific methods etc. I don't think that's the ideal pattern for doing that kind of thing, but it seems like if folks want to do that, the wrapper is helpful.

Reply:
I switched to mount without `render` and interestingly that broke the Vue wrapper assertion in some way. I didn't debug that though, and just switched to your other suggested pattern, which works fine.